Api/Models/NotificationNewFriendRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Reservant.Api.Dtos.User;
+
+namespace Reservant.Api.Models
+{
+    public class NotificationNewFriendRequest : NotificationDetails
+    {
+        public required int FriendRequestId { get; init; }
+        public required DateTime DateRequestSend { get; init; }
+        public required string SenderId { get; init; }
+        public required string ReveiverId { get; init; }
```

Comment:
Receiver to będzie zawsze użytkownik otrzymujący to powiadomienie, więc nie ma potrzeby w ReceiverId